specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3956,6 +4152,329 @@
       },
       "description": "Response from a List Indexes request. If successful, it includes the full definitions of all indexes."
     },
+	"Skillset": {
+		"properties": {
+			"name": {
+				"externalDocs": {
+					"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
+				},
+				"type": "string",
+				"description": "The name of the skillset."
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skillset."
+			},
+			"skills": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/Skill"
+				},
+				"description": "A list of skills in the skillset."
+			},
+			"@odata.etag": {
+				"x-ms-client-name": "ETag",
+				"type": "string",
+				"description": "The ETag of the skillset."
+			}
+		},
+		"required": ["name",
+		"description",
+		"skills"],
+		"description": "A list of Cognitive Search skills."
+	},
+	"Skill": {
+		"discriminator": "@odata.type",
+		"properties": {
+			"@odata.type": {
+				"type": "string"
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skill."
```

Comment:
Descriptions should do more than just repeat the name of the property. They should briefly answer the questions: What is this property? What does it do? When should I use it?

Reply:
Changed to: "description": "The description of the skill which describes the inputs, outputs, and usage of the skill." Thanks.